go.mod

```diff
@@ -1,13 +1,14 @@
 module github.com/luno/reflex
 
-go 1.21
```

Comment:
To support `for i := range N`.

Reply:
Any reason why we can't go for 1.23? If people are still using 1.22, they'd just have to use the version before this update

Reply:
No reason, just that I didn't need it.